src/configparser.js

```diff
@@ -10,14 +10,19 @@ const interpolation = require('./interpolation');
  * @type {RegExp}
  * @private
  */
-const SECTION = new RegExp(/^\s*\[([^\]]+)]$/);
+const SECTION = new RegExp(/^(?<indent>\s*)\[(?<sectionName>[^\]]+)]$/);

```

Comment:
I switch the RegExp to use named groups, so that it's easier to reference them later in the code (vs referencing by index number)